azure-pipelines.yml

```diff
@@ -36,38 +36,66 @@ jobs:
     # test when we're not in a unicode locale
       py310-Clocale:
         python.version: '3.10'
+        python.architecture: 'x64'
         TOXENV: py310-test-deps
         TOX_TESTENV_PASSENV: LANG LC_ALL
         LANG: C
         LC_ALL: C
         H5PY_ENFORCE_COVERAGE: yes
-    # Build HDF5 1.10 or 1.12 in the test environment
+    # Build HDF5 1.10, 1.12 or 1.14 in the test environment
       py310-deps-hdf51107:
         python.version: '3.10'
+        python.architecture: 'x64'
         TOXENV: py310-test-deps
         HDF5_VERSION: 1.10.7
         HDF5_DIR: $(HDF5_CACHE_DIR)/$(HDF5_VERSION)
         H5PY_ENFORCE_COVERAGE: yes
       py310-deps-hdf51122:
         python.version: '3.10'
+        python.architecture: 'x64'
         TOXENV: py310-test-deps
         HDF5_VERSION: 1.12.2
         HDF5_DIR: $(HDF5_CACHE_DIR)/$(HDF5_VERSION)
         H5PY_ENFORCE_COVERAGE: yes
       py310-deps-hdf51146:
         python.version: '3.10'
+        python.architecture: 'x64'
         TOXENV: py310-test-deps
         HDF5_VERSION: 1.14.6
         HDF5_DIR: $(HDF5_CACHE_DIR)/$(HDF5_VERSION)
         H5PY_ENFORCE_COVERAGE: yes
     # do mpi tests
       py310-deps-hdf51122-mpi:
         python.version: '3.10'
+        python.architecture: 'x64'
         TOXENV: py310-test-mindeps-mpi
         HDF5_VERSION: 1.12.2
         HDF5_DIR: $(HDF5_CACHE_DIR)/$(HDF5_VERSION)
         HDF5_MPI: ON
         CC: mpicc
+    # Latest python version
+      py313-deps-hdf51146:
+        python.version: '3.13'
+        python.architecture: 'x64'
+        TOXENV: py313-test-deps
+        HDF5_VERSION: 1.14.6
+        HDF5_DIR: $(HDF5_CACHE_DIR)/$(HDF5_VERSION)
+        H5PY_ENFORCE_COVERAGE: yes
+    # Free-threading (noGIL) Python
+      py313t-deps-hdf51146:
+        python.version: '3.13'
+        python.architecture: 'x64-freethreaded'
```

Comment:
I don't see this value mentionned on https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/use-python-version-v0, could you point me to where you found this setting ?